gitlab-pipeline/stage/post.yml

```diff
@@ -25,6 +25,7 @@ mender-qa:failure:
   # See https://docs.coveralls.io/parallel-build-webhook
   variables:
     COVERALLS_WEBHOOK_URL: "https://coveralls.io/webhook"
+    COVERALLS_TOKEN: ${MENDER_COVERAGE_TOKEN}
```

Comment:
https://github.com/mendersoftware/mender-qa/blob/dc5e156eab67d908eee650c99084498abab2d805/gitlab-pipeline/stage/post.yml#L52 ?

Reply:
Yes, the variable should only be defined in the final jobs, not in the template.

Reply:
I just left this out for now. If it returns I'll look at it again.